.gitignore

```diff
@@ -19,6 +19,8 @@ artifacts
 # make stress, acceptance produce stress.test, acceptance.test
 *.test*
 
+.idea/
```

Comment:
I think we don't want editor-specific stuff in the `.gitignore`, if you look at the top of this file there's instructions for how to add it to your global config.